### DISCUSSION TITLE
未來如何分叉

### DISCUSSION THREAD

**Discussion by Xunhaoz** [1101-GS4719-C/Python-Community-GS4719-C#1336] [Ideas]
我預感到上面諸如“數字雅典”這類技術樂觀主義的內容會招致批評——難道就沒想過反烏托邦的可能嗎？現實一點看，極端的技術烏托邦和反烏托邦應該都不會出現，更可能出現的是凱文·凱利所說的那種“進托邦”（Protopia）——一個逐漸進步的社會，至少是物質生存條件上逐漸進步的社會，精神另說。這個進程已經持續了兩千年，近兩三百年更是急轉直上，沒有理由認為它會突然急轉直下。

全民基本收入是一個美好的願景，我相信這是全球左派精英的共同目標，奧特曼本人不是個單純的技術宅，他早年就寫過關於萬物摩爾定律的文章，人工智能讓未來的生存物資變得極其便宜，結合基本收入，在不工作的情況下，每個人的基本生活都會得到保障。

我嚮往人工智能技術發展帶來全球北歐化甚至全球雅典化的美好前景，也不排除銀翼殺手2049、賽博朋克2077的可能性。但無論未來如何分叉，不影響我今天的這番推論的合理性——親近價值內核的人，能發明自己的遊戲的人，永遠有競爭力，無論未來需不需要競爭。不是說未來是一個樂者為王的時代，今天和過去一直是一個樂者為王的時代。

放棄對“工作”的執念，放棄希望自己是個對社會“有用”的人，那些是舊時代的教條。人生來就是無用的，人類精英都是些born to play，just for fun的大號兒童。很多人沒有搞清楚一件事，真正稀缺的競爭力是錨定一個領域自己卷自己卷出來的，不是咬牙切齒青筋暴起對抗人性犧牲一切苦逼兮兮求出來的。

苦求有用的人只能廝混於外圍，熱衷無用的人才能創造文明。

至此，我希望我描述了一些變化，也揭示出了一些不變的東西。該如何面對今天這場巨變？面對新技術，保持開放和好奇，積極嘗試，但不要過分焦慮，自亂陣腳，要守住那些更恆常的東西。

人們總想要一些具體的行動建議，行不行動另說，只是抄下來也能緩解焦慮……我沒有具體建議，只有一個基本判斷可以分享給你，這個判斷也許能緩解/增加你的焦慮：

如果你本來就是一個有競爭力的人，勤奮、敏感、真誠、自主、善於反思， AI不會讓你喪失競爭力；如果你本來就不具備競爭力，是左顧右盼盲目鑽營的機會主義者，AI也不會讓你具備競爭力。未來的AI就像火與電，人人都能隨取隨用，它會削平一切虛妄的起伏，而那些深深挖洞的人不在乎。

**Comment by hardtagottaname** [comment]
我同意筆者的看法，說到底，AI也只是我們的一個工具，我們不能過度仰賴AI帶給我們的便利而喪失自我思考的能力。拿我自己使用現在很夯的ChatGpt的經驗來說，我問了ChatGpt一個作業問題，但ChatGpt的解法卻和真正答案有所出入，甚至相差甚遠，故我認為ChatGpt所言不一定全為真，我們必須要經過自己的思考，不能過於依賴人工智慧。
